resources/case_studies/Insurance/Bank.obs

```diff
@@ -1,12 +1,22 @@
+// import "resources/case_studies/Insurance/Money.obs"
+import "resources/case_studies/Insurance/Token.obs"
+
+
 main contract Bank {
-    Money mon;
+    //TODO: owned Money mon;
+    int key;
 
     Bank() {
+        // TODO: generate key
```

Comment:
I expect you'll need a BankFactory to issue these? What properties do you need of them?